vumi/components/message_store_migrators.py

```diff
@@ -83,7 +87,7 @@ def migrate_from_1(self, mdata):
         mdata.set_value('$VERSION', 2)
         self._copy_msg_field('msg', mdata)
         mdata.copy_values('batches')
-        mdata.copy_indexes('batches')
+        mdata.copy_indexes('batches_bin')
```

Comment:
This was a pre-existing bug that wasn't picked up because the old indexes were all copied anyway?